test/Elastica/Aggregation/GeoBoundsTest.php

```diff
@@ -8,7 +8,7 @@
 
 class GeoBoundsTest extends BaseAggregationTest
 {
-    protected function _getIndexForTest()
+    private function getIndexForTest()
```

Comment:
Sounds good. But IMO we should only combine it with some other changes. Only  renaming properties does not justify a BC break.